WifiART/src/structs/LciState.java

```diff
@@ -196,6 +202,9 @@ public void setLciVersion(int lciVersion) {
      * @param latitude the latitude, in degrees
      */
     public void setLatitude(double latitude) {
+        if (latitude < MIN_LATITUDE || latitude > MAX_LATITUDE) {
+            throw new NumberFormatException();
```

Comment:
nit: consider adding more details in the exception - something like "latitude xxx is out of range [-90, 90]", so it's clearer what went wrong.

Reply:
I currently have the convention of defining my error messages in the controller, since the controller acts as an intermediary between the model and the view. I might change this though, because I agree that it would make sense to define the error where it occurs.

Reply:
I second having more explanatory exception names.